src/main/java/org/broadinstitute/consent/http/service/DataAccessRequestService.java

```diff
@@ -297,11 +297,12 @@ public DataAccessRequest createProgressReport(User user, DataAccessRequest progr
       }
     }
 
+    syncDataAccessRequestDatasets(progressReportDatasetIds, referenceId);
```

Comment:
ruleService.triggerDACRuleSettings is dependent upon the dar_dataset relationships being memorialized so that emails can be sent out.